lib/ruby_motion_query.rb

```diff
@@ -11,4 +11,9 @@
   files << Dir.glob(File.join(parent, "motion/**/*.rb"))
   files.flatten!.uniq!
   app.files.unshift files
+
+  app.development do
```

Comment:
This is okay the BuildTime, but not for the RootPath, if we put those lines in lib, `__FILE__` will become lib, instead of the actual project root. That's why I closed this PR.